README.md

```diff
@@ -1,284 +1,126 @@
 
-1. Configure the project to use the Edge Runtime:
+```sh
+npx create-next-app@13.2.4 my-next-app
+```
 
-   Replace `pages/api/hello.ts` with the following:
+<details>
 
-   ```typescript
-   // Next.js Edge API Routes: https://nextjs.org/docs/api-routes/edge-api-routes
-   import type { NextRequest } from 'next/server';
+<summary>Note on the Next.js version</summary>
 
-   export const config = {
-   	runtime: 'experimental-edge',
-   };
+The above command suggests to use Next's 13.2.4 version, that is the latest at the time of writing, older versions are also supported (but they might be only partially), newer versions might not be so be wary of that. The team will soon test various versions and document more clearly which are supported to what degree.
 
-   export default async function handler(req: NextRequest) {
-   	return new Response(JSON.stringify({ name: 'John Doe' }), {
-   		status: 200,
-   		headers: {
-   			'Content-Type': 'application/json',
-   		},
-   	});
-   }
-   ```
+</details>
 
-1. `git commit` and `git push` to a GitHub/GitLab repository.
+&NewLine;
 
-1. Create a Pages project, connect that repository, and select "Next.js" from the framework preset list.
+Change your current directory to the newly created one, as the following step require you to be in it:
 
-   | Option                 | Value                                                 |
-   | ---------------------- | ----------------------------------------------------- |
-   | Build command          | `npx @cloudflare/next-on-pages --experimental-minify` |
-   | Build output directory | `.vercel/output/static`                               |
+```sh
+cd my-next-app
+```
 
-1. Add a `NODE_VERSION` environment variable set to `16` or greater (`18` is not supported yet, See [Build Image Update Discussion](https://github.com/cloudflare/pages-build-image/discussions/1).
+### 2. Configure the application to use the Edge Runtime
 
-1. In the Pages project **Settings** > **Functions** > **Compatibility Flags**, add the `nodejs_compat` and ensure the **Compatibility Date** is set to at least `2022-11-30`.
+In order for the application to run on Cloudflare Pages it needs to be set to run on the edge runtime, in order to do so make sure that all the files in your application containing server side code (as your api routes and pages using `getServerSideProps`) export a config object specifying the use of the edge runtime:
 
-1. The project should now be ready to deploy. Create a new deployment.
+```ts
+export const config = {
+	runtime: 'edge',
+};
+```
 
-## `@cloudflare/next-on-pages` CLI
+Make also sure that your application is not using [unsupported APIs](https://nextjs.org/docs/api-reference/edge-runtime#unsupported-apis) and its API routes are defined as [Edge API Routes](https://nextjs.org/docs/api-routes/edge-api-routes).
 
-```
-⚡️ @cloudflare/next-to-pages CLI
```

Comment:
I don't think showing the help message is very valuable since the developer can easily see it using `--help`, showing the message here also has the risk of this getting out of sync with the current help message, so I changed this section to: `If you provide the '--help' flag a useful help message will be displayed showing you the various options the CLI offers.`